.github/workflows/distribution-version.py

```diff
@@ -0,0 +1,89 @@
+import os
+import re
+
+import bb.tinfoil
+import git
+
+# TODO: add minor version numbers once we have them
+RE_VERSION_TAG = "v(?P<year>[0-9][0-9])\.(?P<month>[0-9][0-9])"
+RE_VERSION_BB = "(?P<year>[0-9][0-9])\.(?P<month>[0-9][0-9])(?P<dev>$|\+dev)"
+
+
+def bb_variables(names):
+    ret = dict()
+
+    with bb.tinfoil.Tinfoil(tracking=True, setup_logging=True) as tinfoil:
+        tinfoil.prepare(quiet=2, config_only=True)
+        d = tinfoil.config_data
+
+        for name in names:
+            ret[name] = d.getVar(name)
+
+    return ret
+
+
+def git_prev_tag():
+    repo = git.Repo()
+
+    current_commit_is_tagged = True
+
+    for commit in repo.iter_commits():
+        for tag in repo.tags:
+            if tag.commit == commit:
+                return (tag, current_commit_is_tagged)
+
+        current_commit_is_tagged = False
+
+
+def check_version(distro_version):
+    prev_tag, commit_is_tagged = git_prev_tag()
+
+    print(f"Checking tag {prev_tag.name} against version {distro_version}")
+
+    version_tag = re.fullmatch(RE_VERSION_TAG, prev_tag.name)
+    version_tag_numeric = int(version_tag["year"]) * 100 + int(version_tag["month"])
+
+    version_bb = re.fullmatch(RE_VERSION_BB, distro_version)
+    version_bb_numeric = int(version_bb["year"]) * 100 + int(version_bb["month"])
+
+    if commit_is_tagged:
+        # The version in a tagged commit must match the version in the tag's name.
+        assert version_bb["dev"] == ""
+        assert version_tag_numeric == version_bb_numeric
+
+    elif version_bb["dev"] == "":
+        # Release candidates already have the next release version set,
+        # but there must not be a tag with the same or a newer version number
+        # in the current commits history.
```

Comment:
```suggestion
        # Release candidates already have the next release version set,
        # but it must be newer than any tag in the current commit's history.
```

Reply:
Yep, that's a less round-about description. I've copied the recommendation into my updated PR.